<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>983</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>监听端口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBoxAddress">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btnSend">
      <property name="text">
       <string>发送信息</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTextEdit" name="textEditSendMsg"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelAddress">
      <property name="text">
       <string>监听地址</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="minimum">
       <number>1025</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionListening">
   <property name="text">
    <string>开始监听</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>停止监听</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空文本框</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>加粗</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>倾斜</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>设置字体</string>
   </property>
  </action>
  <action name="actionFontSize">
   <property name="text">
    <string>设置字体大小</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>添加图片</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>查看当前样式</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="actionHorCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>水平居中</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>上对齐</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>下对齐</string>
   </property>
  </action>
  <action name="actionVerCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>垂直居中对齐</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
